### PR TITLE
Adopt AIP-205: Beta-blocking changes

### PR DESCRIPTION
**Pull request by rofrankel**

### CODE REVIEW

File: aep/general/0205/aep.yaml [hunk 1]
```diff
@@ -2,6 +2,6 @@
 id: 205
 state: reviewing
```
Comment (toumorokoshi) [inline aep/general/0205/aep.yaml:3]:
```suggestion
state: approved
```

File: aep/general/0205/aep.md.j2 [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+  // Parameters for input.
+  // (-- aep.dev/beta-blocker: Convert well-known parameters into explicit
+  //     fields before the Beta launch. --)
+  map<string, string> parameters = 1;
+}
+```
+
+{% tab oas %}
+
+**Note:** OAS example not yet written.
```
Comment (toumorokoshi) [inline aep/general/0205/aep.md.j2:42]: can you file an issue on these so we remember where they are? folks may be interested in picking these up as well, post barn-raising.

File: aep/general/0205/aep.md.j2 [hunk 3]
```diff
@@ -1,5 +1,47 @@
 # Beta-blocking changes
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/50) for more
-information.
+APIs often release an Alpha version of their API in order to get early feedback
```
Comment (toumorokoshi) [inline aep/general/0205/aep.md.j2:3]: should we capitalize alpha?

File: aep/general/0205/aep.md.j2 [hunk 4]
```diff
@@ -1,5 +1,47 @@
...
+important feedback is incorporated and the API is made stable for Beta.
+
+Since the purpose of Alpha is to gather feedback, the API does not need to be
+perfect yet, and it's not strictly necessary for API authors to address every
+usability concern or address every point in the API standards. Often, API
+authors and API reviewers will not agree on the best design, and the best way
+to find out is by having users try out the API.
+
+However, once the feedback has been collected and the API is going to be
+promoted to Beta, usability concerns and style issues do need to be addressed.
```
Comment (toumorokoshi) [inline aep/general/0205/aep.md.j2:14]:
```suggestion
promoted to Beta, usability concerns and style issues need to be addressed.
```
removing superfluous words.

File: aep/general/0205/aep.md.j2 [hunk 5]
```diff
@@ -1,5 +1,47 @@
...
+In order to ensure that these issues are not forgotten, they should be
```
Comment (toumorokoshi) [inline aep/general/0205/aep.md.j2:15]:
```suggestion
In order to ensure that these issues are not forgotten, they **should** be
```

File: aep/general/0205/aep.md.j2 [hunk 6]
```diff
@@ -1,5 +1,47 @@
...
+explicitly documented in the API.
+
+## Guidance
+
+If an API has usability concerns or violates API standards, and the present
+design should receive additional scrutiny before being carried through to the
+Beta version, there **must** be an internal comment linking to this document
+using its descriptive link ([aep.dev/beta-blocker]()) to ensure that the design
```
Comment (toumorokoshi) [inline aep/general/0205/aep.md.j2:23]:
```suggestion
using its descriptive link ([aep.dev/beta-blocking-changes]()) to ensure that the design
```

Comment (toumorokoshi) [reply aep/general/0205/aep.md.j2:23]: or maybe `aep.dev/beta-blocker` could be a `redirect_from: /beta-blocker` in the yaml file?

**Review by toumorokoshi — CHANGES_REQUESTED** [review]
mostly good! only thing I feel like we need to do is have `aep.dev/beta-blocker` redirect somewhere with an explanation. Otherwise LGTM.

**Comment by earth2marsh** [comment]
I wonder if AEPs should have an opinion on the lifecycle of changes. Alpha/Beta is a fairly common approach, but is it really necessary for AEPs to adopt this AIP?

File: aep/general/0205/aep.md.j2 (hunk 2 above)
Comment (rofrankel) [reply aep/general/0205/aep.md.j2:42]: Yes - but if you're okay with it, I'd like to do it as a batch process (to make sure I catch everything) decoupled from any specific PR.

File: aep/general/0205/aep.md.j2 (hunk 3 above)
Comment (rofrankel) [reply aep/general/0205/aep.md.j2:3]: I don't think so, fixed throughout.

File: aep/general/0205/aep.md.j2 (hunk 2 above)
Comment (toumorokoshi) [reply aep/general/0205/aep.md.j2:42]: sure, but I think we have to be reasonable about the "batches" - it's unlikely that someone is going to do every single missing OAS example in one PR.

Can you make it granular to the AEP? IF you file an issue go ahead and resolve.

**Review by toumorokoshi — APPROVED** [review]
thanks! I figure you'll file that issue so no blocking comments.

**Comment by toumorokoshi** [comment]
BTW note the lint error that's blocking auto-merge.

File: aep/general/0205/aep.md.j2 [hunk 7]
```diff
@@ -1,5 +1,47 @@
-# Beta-blocking changes
+# beta-blocking changes
```
Comment (mkistler) [inline aep/general/0205/aep.md.j2:1]: Why this change? Seems odd for an h1 to not start with a capital.

**Review by mkistler — APPROVED** [review]
Looks good. 👍 

But I think the title should be fixed.

File: aep/general/0205/aep.md.j2 (hunk 7 above)
Comment (rofrankel) [reply aep/general/0205/aep.md.j2:1]: Wasn't deliberate; fixed.
